styles/design-settings/cardboard/_design.scss

```diff
@@ -24,7 +24,9 @@
 @import 'designs/cardboard/parts/notes/notes-iconCondensed-shapes.scss';
 @import 'designs/cardboard/parts/index-shapes';
 @import 'designs/cardboard/parts/exercises-shapes';
-@import 'designs/cardboard/parts/lists-shapes';
+@import 'designs/cardboard/parts/lists/lists-from-elements-shapes';
+@import 'designs/cardboard/parts/lists/lists-from-modules-shapes';
+@import 'designs/cardboard/parts/lists/lists-with-attributes-shapes';
```

Comment:
I wasn't sure how to name these files. If you have any better ideas, something that might be more descriptive, please let me know :wink:

Reply:
These look self-explanatory to me!